res/TemplateBatchFiles/SelectionAnalyses/FEL.bf

```diff
@@ -104,13 +109,13 @@ namespace fel {
 
 
 /* Prompt for one-rate or two-rate analysis */
-fel.tworate = io.SelectAnOption( {{"Yes", "[Recommended] Consider synonymous rate variation (dS varies across sites)."}, {"No", "Ignore synonymous rate variation (dS := 1 at each site)."}},
+fel.srv = io.SelectAnOption( {{"Yes", "[Recommended] Consider synonymous rate variation (dS varies across sites)."}, {"No", "Ignore synonymous rate variation (dS := 1 at each site)."}},
                                   "Use synonymous rate variation? Strongly recommended YES for selection inference.");
-console.log(fel.tworate);
-if (fel.tworate == "Yes"){
-    fel.tworate = TRUE
+console.log(fel.srv);
```

Comment:
It might not be a bad idea to see if an es6 linter could be adapted to use with HBL to check for stray console.logs.